### PR TITLE
DBT-811 Feat/lesson attendes

### PR DESCRIPTION
**Pull request by kayiganwa04**
## Context

here we are adding lesson attendees to see who is attending this lesson

[Lesson Attendees](https://www.notion.so/tianlu/Frontend-Lesson-Attendees-004905f5e05148c5b1fa3f8a4f56df0a?pvs=4)

## Test

- Click on the lesson information
- On the lesson information the is a lesson attendee which will direct you to the lessons attendes page

### CODE REVIEW

File: src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      this.$refs.addStudents.open()
+    },
+    async filterByWillAssist() {
+      this.willAssist = !this.willAssist
+      this.loadLessonStudents()
+      console.log(this.willAssist)
+    },
+    deleteGroupFromLesson() {
+      this.$refs.deleteGroupModal.open()
+    },
```
Comment (abelbordonado) [inline src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue:123]: No need

File: src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  methods: {
+    ...mapActions('lessonAttendeesStore', ['resetTableOptions']),
+    ...mapMutations('lessonAttendeesStore', ['SET_TABLE_OPTIONS']),
+    addStudents() {
+      this.$refs.addStudents.open()
+    },
+    async filterByWillAssist() {
+      this.willAssist = !this.willAssist
+      this.loadLessonStudents()
+      console.log(this.willAssist)
```
Comment (abelbordonado) [inline src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue:119]: 🦞

File: src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        cancelButtonText: 'Cancelar'
+      })
+
+      if (!result.isConfirmed) {
+        return
+      }
+      const res = await LessonRepository.deleteStudentFromLesson(lessonId, {
+        student_id
+      })
+
```
Comment (abelbordonado) [inline src/modules/lessons/lesson-attendees/components/lesson-attendees-list.vue:167]: We can not delete from here

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
https://academy-750.slack.com/archives/C05JEU4AP5Y/p1695833299177529

image


Remove horizontal scroller from calendar view in students / admin

**Comment by kayiganwa04** [comment]
> image



> image

@abelbordonado this one I don't get which reducer you want me to reset

**Pull request merged**